python_modules/dagster/dagster_tests/core_tests/asset_defs_tests/test_asset_graph.py

```diff
@@ -27,14 +40,21 @@ def asset2(asset0):
     def asset3(asset1, asset2):
         ...
 
-    asset_graph = AssetGraph.from_assets([asset0, asset1, asset2, asset3])
-    assert asset_graph.all_asset_keys == {asset0.key, asset1.key, asset2.key, asset3.key}
-    assert not asset_graph.is_partitioned(asset0.key)
-    assert asset_graph.is_partitioned(asset1.key)
-    assert asset_graph.have_same_partitioning(asset1.key, asset2.key)
-    assert not asset_graph.have_same_partitioning(asset1.key, asset3.key)
-    assert asset_graph.get_children(asset0.key) == {asset1.key, asset2.key}
-    assert asset_graph.get_parents(asset3.key) == {asset1.key, asset2.key}
+    assets = [asset0, asset1, asset2, asset3]
```

Comment:
this is great, I meant to get around to this -- any reason `internal_asset_graph == external_asset_graph` would be expected to evaluate to False at this point?